cloud/google/cmd/gce-machine-controller/Makefile

```diff
@@ -18,7 +18,7 @@ GCR_BUCKET = k8s-cluster-api
 PREFIX = gcr.io/$(GCR_BUCKET)
 DEV_PREFIX ?= gcr.io/$(shell gcloud config get-value project)
 NAME = gce-machine-controller
-TAG = 0.0.9
+TAG = 0.0.11
```

Comment:
Sorry.. looks like 0.0.11 is also taken now by #213.

Reply:
NP, updated to 0.0.12.